frontend/views/pages/GroupDashboard.vue

```diff
@@ -2,7 +2,7 @@
 page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='groupSettings.groupName')
   template(#title='') {{ groupSettings.groupName }}
 
-  add-income-details-widget(v-if='!hasIncomeDetails')
+  add-income-details-widget(v-if='!hasIncomeDetails' :welcomeMessage='true')
```

Comment:
rename to `hasWelcomeMessage`, so it's more intuitive that's a Boolean, not a String.